Assets/Scripts/PauseMenu.cs

```diff
@@ -0,0 +1,38 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class PauseMenu : MonoBehaviour
+{
+    public static bool PAUSED = false;
+    public GameObject pauseUI;
+
+    // Update is called once per frame
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Escape)) {
```

Comment:
Invert if and return to reduce nesting

Reply:
I don't like using return in that way in the update function. If I want other behaviors added later that will prevent it.